.github/workflows/build.yml

```diff
@@ -100,6 +107,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
+      # Free GitHub Actions Environment Disk Space
+      - name: Maximize Build Space
+        uses: jlumbroso/free-disk-space@v1.3.1
```

Comment:
I can confirm that this reconfiguration works! 🙌

Why do we point to a specific version instead of the `jlumbroso/free-disk-space@main` ref?

Reply:
> Why do we point to a specific version instead of the `jlumbroso/free-disk-space@main` ref?

It's generally a good idea to pin actions to avoid unexpected changes. And actually, since this is a third-party action not maintained by JetBrains, it would be best to pin it to a particular SHA to [improve security](https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-third-party-actions:~:text=Pinning%20to%20a%20particular%20SHA%20helps%20mitigate%20the%20risk%20of%20a%20bad%20actor%20adding%20a%20backdoor%20to%20the%20action%27s%20repository). (`jlumbroso/free-disk-space@54081f138730dfa15788a46383842cd2f914a1be` is v1.3.1.)